src/copilot/inspect/index.ts

```diff
@@ -0,0 +1,32 @@
+import { CancellationToken, CodeAction, CodeActionContext, CodeActionKind, ExtensionContext, TextDocument, languages, window, workspace, Range, Selection } from "vscode";
+import { COMMAND_INSPECT_RANGE, registerCommands } from "./commands";
+import { InspectActionCodeLensProvider } from "./InspectActionCodeLensProvider";
+
+export const DEPENDENT_EXTENSIONS = ['github.copilot-chat', 'redhat.java'];
+
+export async function activateCopilotInspection(context: ExtensionContext): Promise<void> {
+
+    registerCommands();
+
+    const inspectActionCodeLenses = new InspectActionCodeLensProvider().install(context);
+
+    context.subscriptions.push(
+        workspace.onDidOpenTextDocument(doc => inspectActionCodeLenses.rerender(doc)), // Rerender class codelens when open a new document
+        workspace.onDidChangeTextDocument(e => inspectActionCodeLenses.rerender(e.document)), // Rerender class codelens when change a document
```

Comment:
do you think it's necessary to debounce the onDidChangeTextDocument event?

Reply:
good point, it's debounced in later PRs.